.github/PULL_REQUEST_TEMPLATE/new_feature.md

```diff
@@ -0,0 +1,9 @@
+## This PR adds...
+
+*List your features here and their reasons for creation*
+
+## Notes
+
+*List anything note-worthy here (potential issues, this needs merge to `master` before working etc...)*
```

Comment:
One extra dot needed, because `etc.`. 🙂
```suggestion
*List anything note-worthy here (potential issues, this needs merge to `master` before working etc....)*
```